acceptance/detector_test.go

```diff
@@ -349,6 +349,11 @@ fail: fail_detect_buildpack@some_version
 			os.RemoveAll(copyDir)
 		})
 
+		experimentalMode := "warn"
+		if api.MustParse(platformAPI).AtLeast("0.13") {
```

Comment:
Ah, for the detector tests we only exercise the "latest" Platform API version, hence `undefined: platformAPI` - we can just set the var to error always

Reply:
Done!